tests/util.py

```diff
@@ -118,6 +115,70 @@ def make_fcc(nx=1, ny=1, nz=1, scale=1.0, noise=0.0):
     return freud.box.Box(*box), positions
 
 
+def make_fcc(nx=1, ny=1, nz=1, scale=1.0, noise=0.0):
+    """Make a FCC crystal for testing
+
+    Args:
+        nx: Number of repeats in the x direction, default is 1
+        ny: Number of repeats in the y direction, default is 1
+        nz: Number of repeats in the z direction, default is 1
+        scale: Amount to scale the unit cell by (in distance units)
+               (Default value = 1.0).
+        noise: Apply Gaussian noise with this width to particle positions
+               (Default value = 0.0).
+
+    Returns:
+        tuple (py:class:`freud.box.Box`, :class:`np.ndarray`): freud Box,
+            particle positions, shape=(nx*ny*nz, 3)
```

Comment:
Probably should be `4*nx*ny*nz` but maybe there's a better way to write that across all of these functions.

Reply:
It's fine, I'll just add it this way. It's a util helper, so more important to have the information than have it in a way that renders nicely.